Example-L476/blink476/Core/Inc/stm32l4xx_hal_conf.h

```diff
@@ -0,0 +1,483 @@
+/**

```

Comment:
So I don't think we will want to track this in git, see the other Blink project gitignores that live in Src and Inc that ignore these generated files